app/liquid_tags/github_tag/github_issue_tag.rb

```diff
@@ -91,7 +91,7 @@ def finalize_html(input)
     def valid_link?(link)
       link_without_domain = link.gsub(/.*github\.com\//, "").split("/")
       raise_error unless [
-        !!(link =~ /.*github\.com\//),
```

Comment:
Here `link =~ /.*github\.com\//`does not return the same result as `!!(link =~ /.*github\.com\//)`. The `!!` returns boolean and the new version returns an integer or `nil`. Here `!!` is used to convert the returning value to boolean, so I believe it should be kept as it was (at least for now) + the rule should be disabled by `Exclude` in config or disabled for this block of code.
You can also run a test suite before committing (`bundle exec rspec spec/`), some of the errors can be caught earlier by doing that.